components/datetime/src/skeleton.rs

```diff
@@ -722,6 +723,42 @@ mod test {
         };
     }
 
+    // TODO(#586) - Append items support needs to be added.
+    #[test]
```

Comment:
@zbraniecki - I added this test since I didn't know what the behavior was here. It turns out the time zone fields are not even added by the components bag yet. I've found it helpful working incrementally to have tests like these that show when something is correctly implemented.